src/Qir/Runtime/lib/QIR/QubitManager.cpp

```diff
@@ -92,15 +92,15 @@ CQubitManager::QubitIdType CQubitManager::QubitListInSharedArray::TakeQubitFromF
 
     if (result != NoneMarker)
     {
-        sharedQubitStatusArray[result] = AllocatedMarker;
+        sharedQbitStatusArray[result] = AllocatedMarker;
     }
 
     return result;
 }
 
-void CQubitManager::QubitListInSharedArray::MoveAllQubitsFrom(QubitListInSharedArray& source, QubitIdType* sharedQubitStatusArray)
+void CQubitManager::QubitListInSharedArray::MoveAllQubitsFrom(QubitListInSharedArray& source, QubitIdType* sharedQbitStatusArray)
```

Comment:
Same here, should it be `sharedQubitStatusArray` instead?

Reply:
Resolved in a different discussion.